content/en/docs/v3.5/op-guide/configuration.md

```diff
@@ -4,7 +4,7 @@ weight: 4050
 description: etcd configuration files, flags, and environment variables
 ---
 
-You can configure etcd through the following:
+You can configure etcd through either of the following (or a combination of those):
```

Comment:
`either` refers to two items, there are three items in the list so that term doesn't really work. We would need to say "one of the following" or something similar.

When using a configuration file it is not possible to combine with environment variables or command flags.

Reply:
I see. how about at least accepting the proposed `either of` (which was my main point to be honest)?

Reply:
The problem is grammatically [either](https://dictionary.cambridge.org/grammar/british-grammar/either) refers to two choices, when in our list there are three options available. 

I suggest we keep the current wording but please let me know if I am misunderstanding something.